### ISSUE TITLE
Request failed with status code 429

### ISSUE THREAD

**Issue by fachos** [iptv-org/epg#2361]
### Describe your issue

Hi,

I try to grab https://content.astro.com.my
after some time it flood with code 429

here is some logs

```
> grab
> npx tsx scripts/commands/epg/grab.ts --site=content.astro.com.my

(node:24859) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("file%3A///opt/epg-2023.12.1/node_modules/tsx/dist/loader.mjs", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
starting...
config:
  output: guide.xml
  maxConnections: 1
  gzip: false
  site: content.astro.com.my
loading channels...
  found 157 channel(s)
run #1:
  [1/314] content.astro.com.my (ms) - 14 - Apr 23, 2024 (0 programs)
  [2/314] content.astro.com.my (ms) - 14 - Apr 24, 2024 (6 programs)
  [3/314] content.astro.com.my (ms) - 15 - Apr 23, 2024 (0 programs)
  [4/314] content.astro.com.my (ms) - 15 - Apr 24, 2024 (6 programs)
  [5/314] content.astro.com.my (ms) - 17 - Apr 24, 2024 (5 programs)
  [6/314] content.astro.com.my (ms) - 17 - Apr 23, 2024 (0 programs)
  [7/314] content.astro.com.my (ms) - 16 - Apr 24, 2024 (5 programs)
  [8/314] content.astro.com.my (ms) - 16 - Apr 23, 2024 (0 programs)
  [9/314] content.astro.com.my (ms) - 18 - Apr 24, 2024 (5 programs)
  [10/314] content.astro.com.my (ms) - 18 - Apr 23, 2024 (0 programs)
  [11/314] content.astro.com.my (ms) - 21 - Apr 24, 2024 (5 programs)
  [12/314] content.astro.com.my (ms) - 29 - Apr 24, 2024 (46 programs)
  [13/314] content.astro.com.my (ms) - 474 - Apr 24, 2024 (11 programs)
  [14/314] content.astro.com.my (ms) - 474 - Apr 23, 2024 (0 programs)
  [15/314] content.astro.com.my (ms) - 29 - Apr 23, 2024 (0 programs)
  [16/314] content.astro.com.my (ms) - 472 - Apr 24, 2024 (29 programs)
  [17/314] content.astro.com.my (ms) - 472 - Apr 23, 2024 (0 programs)
  [18/314] content.astro.com.my (ms) - 21 - Apr 23, 2024 (0 programs)
  [19/314] content.astro.com.my (ms) - 28 - Apr 24, 2024 (6 programs)
  [20/314] content.astro.com.my (ms) - 467 - Apr 24, 2024 (40 programs)
  [21/314] content.astro.com.my (ms) - 467 - Apr 23, 2024 (0 programs)
  [22/314] content.astro.com.my (ms) - 28 - Apr 23, 2024 (0 programs)
  [23/314] content.astro.com.my (ms) - 461 - Apr 24, 2024 (27 programs)
  [24/314] content.astro.com.my (ms) - 461 - Apr 23, 2024 (0 programs)
  [25/314] content.astro.com.my (ms) - 20 - Apr 24, 2024 (5 programs)
  [26/314] content.astro.com.my (ms) - 27 - Apr 24, 2024 (5 programs)
  [27/314] content.astro.com.my (ms) - 459 - Apr 24, 2024 (0 programs)
  [28/314] content.astro.com.my (ms) - 459 - Apr 23, 2024 (0 programs)
  [29/314] content.astro.com.my (ms) - 27 - Apr 23, 2024 (0 programs)
  [30/314] content.astro.com.my (ms) - 394 - Apr 24, 2024 (0 programs)
  [31/314] content.astro.com.my (ms) - 394 - Apr 23, 2024 (0 programs)
  [32/314] content.astro.com.my (ms) - 20 - Apr 23, 2024 (0 programs)
  [33/314] content.astro.com.my (ms) - 26 - Apr 24, 2024 (6 programs)
  [34/314] content.astro.com.my (ms) - 312 - Apr 24, 2024 (39 programs)
  [35/314] content.astro.com.my (ms) - 312 - Apr 23, 2024 (0 programs)
  [36/314] content.astro.com.my (ms) - 26 - Apr 23, 2024 (0 programs)
  [37/314] content.astro.com.my (ms) - 311 - Apr 24, 2024 (50 programs)
  [38/314] content.astro.com.my (ms) - 311 - Apr 23, 2024 (0 programs)
  [39/314] content.astro.com.my (ms) - 19 - Apr 24, 2024 (5 programs)
  [40/314] content.astro.com.my (ms) - 25 - Apr 24, 2024 (6 programs)
  [41/314] content.astro.com.my (ms) - 310 - Apr 24, 2024 (21 programs)
  [42/314] content.astro.com.my (ms) - 310 - Apr 23, 2024 (0 programs)
  [43/314] content.astro.com.my (ms) - 25 - Apr 23, 2024 (0 programs)
  [44/314] content.astro.com.my (ms) - 303 - Apr 24, 2024 (6 programs)
  [45/314] content.astro.com.my (ms) - 303 - Apr 23, 2024 (0 programs)
  [46/314] content.astro.com.my (ms) - 19 - Apr 23, 2024 (0 programs)
  [47/314] content.astro.com.my (ms) - 24 - Apr 24, 2024 (5 programs)
  [48/314] content.astro.com.my (ms) - 293 - Apr 24, 2024 (8 programs)
  [49/314] content.astro.com.my (ms) - 293 - Apr 23, 2024 (0 programs)
  [50/314] content.astro.com.my (ms) - 24 - Apr 23, 2024 (0 programs)
  [51/314] content.astro.com.my (ms) - 292 - Apr 24, 2024 (6 programs)
  [52/314] content.astro.com.my (ms) - 292 - Apr 23, 2024 (0 programs)
  [53/314] content.astro.com.my (ms) - 487 - Apr 23, 2024 (0 programs)
  [54/314] content.astro.com.my (ms) - 23 - Apr 24, 2024 (5 programs)
  [55/314] content.astro.com.my (ms) - 145 - Apr 24, 2024 (8 programs)
  [56/314] content.astro.com.my (ms) - 486 - Apr 24, 2024 (16 programs)
  [57/314] content.astro.com.my (ms) - 486 - Apr 23, 2024 (0 programs)
  [58/314] content.astro.com.my (ms) - 145 - Apr 23, 2024 (0 programs)
  [59/314] content.astro.com.my (ms) - 485 - Apr 24, 2024 (60 programs)
  [60/314] content.astro.com.my (ms) - 485 - Apr 23, 2024 (0 programs)
  [61/314] content.astro.com.my (ms) - 23 - Apr 23, 2024 (0 programs)
  [62/314] content.astro.com.my (ms) - 32 - Apr 24, 2024 (5 programs)
  [63/314] content.astro.com.my (ms) - 483 - Apr 24, 2024 (29 programs)
  [64/314] content.astro.com.my (ms) - 483 - Apr 23, 2024 (0 programs)
  [65/314] content.astro.com.my (ms) - 32 - Apr 23, 2024 (0 programs)
  [66/314] content.astro.com.my (ms) - 482 - Apr 24, 2024 (29 programs)
  [67/314] content.astro.com.my (ms) - 482 - Apr 23, 2024 (0 programs)
  [68/314] content.astro.com.my (ms) - 22 - Apr 24, 2024 (6 programs)
  [69/314] content.astro.com.my (ms) - 31 - Apr 24, 2024 (5 programs)
  [70/314] content.astro.com.my (ms) - 481 - Apr 24, 2024 (156 programs)
  [71/314] content.astro.com.my (ms) - 481 - Apr 23, 2024 (0 programs)
  [72/314] content.astro.com.my (ms) - 31 - Apr 23, 2024 (0 programs)
  [73/314] content.astro.com.my (ms) - 478 - Apr 24, 2024 (48 programs)
  [74/314] content.astro.com.my (ms) - 478 - Apr 23, 2024 (0 programs)
  [75/314] content.astro.com.my (ms) - 22 - Apr 23, 2024 (0 programs)
  [76/314] content.astro.com.my (ms) - 30 - Apr 24, 2024 (10 programs)
  [77/314] content.astro.com.my (ms) - 477 - Apr 24, 2024 (9 programs)
  [78/314] content.astro.com.my (ms) - 477 - Apr 23, 2024 (0 programs)
  [79/314] content.astro.com.my (ms) - 30 - Apr 23, 2024 (0 programs)
  [80/314] content.astro.com.my (ms) - 476 - Apr 24, 2024 (14 programs)
  [81/314] content.astro.com.my (ms) - 476 - Apr 23, 2024 (0 programs)
  [82/314] content.astro.com.my (ms) - AstroOasis.my - Apr 24, 2024 (53 programs)
  [83/314] content.astro.com.my (ms) - AstroOasis.my - Apr 23, 2024 (0 programs)
  [84/314] content.astro.com.my (ms) - AstroHuaHeeDai.my - Apr 24, 2024 (20 programs)
  [85/314] content.astro.com.my (ms) - AstroHuaHeeDai.my - Apr 23, 2024 (0 programs)
  [86/314] content.astro.com.my (ms) - AstroCricket.my - Apr 24, 2024 (11 programs)
  [87/314] content.astro.com.my (ms) - AstroCricket.my - Apr 23, 2024 (0 programs)
  [88/314] content.astro.com.my (ms) - AstroCitra.my - Apr 24, 2024 (17 programs)
  [89/314] content.astro.com.my (ms) - AstroCitra.my - Apr 23, 2024 (0 programs)
  [90/314] content.astro.com.my (ms) - AstroCeria.my - Apr 24, 2024 (130 programs)
  [91/314] content.astro.com.my (ms) - AstroCeria.my - Apr 23, 2024 (0 programs)
  [92/314] content.astro.com.my (ms) - AstroBoxOfficeThangathirai.my - Apr 24, 2024 (20 programs)
  [93/314] content.astro.com.my (ms) - AstroBoxOfficeThangathirai.my - Apr 23, 2024 (0 programs)
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
  [94/314] content.astro.com.my (ms) - AstroAwani.my - Apr 24, 2024 (60 programs)
  [95/314] content.astro.com.my (ms) - AstroAwani.my - Apr 23, 2024 (0 programs)
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
Request failed with status code 429
```

need help to resolve this issue..

thank you

**Comment by mavijavi5124** [comment]
Got the same issue as well

**Comment by freearhey** [comment]
Closed in favor of #2363
